canvas_modules/common-canvas/src/common-properties/components/table-buttons/table-buttons.jsx

```diff
@@ -123,6 +124,7 @@ TableButtons.propTypes = {
 	propertyId: PropTypes.object.isRequired,
 	customButtons: PropTypes.array.isRequired,
 	tableState: PropTypes.string,
+	toolbarOverflowLabel: PropTypes.string,
```

Comment:
Why does this need to be passed in as a prop?  Does this show up in the UI or is it only an aria-label?

Reply:
This is being used as label to be passed as `aria-label` inside `toolbar-overflow-item.jsx` `Button` element.

https://github.com/elyra-ai/canvas/blob/4a6061a22a96503d29e0660ee8c2275534b7769f/canvas_modules/common-canvas/src/toolbar/toolbar-overflow-item.jsx#L165